lib/util/git/private-key.ts

```diff
@@ -29,17 +29,24 @@ async function importKey(): Promise<void> {
   await fs.remove(keyFileName);
 }
 
-export async function writePrivateKey(cwd: string): Promise<void> {
+export async function writePrivateKey(): Promise<void> {
   if (!gitPrivateKey) {
     return;
   }
   logger.debug('Setting git private key');
   try {
     await importKey();
-    await exec(`git config user.signingkey ${keyId}`, { cwd });
-    await exec(`git config commit.gpgsign true`, { cwd });
```

Comment:
@rarkins Shouldn't we simply set this with `--global` ? So it's written to user home, as the key is imported to user home too. 🤔

Reply:
> @rarkins Shouldn't we simply set this with `--global` ? So it's written to user home, as the key is imported to user home too. 🤔

while `commit.gpgsign` can't do much harm but `user.signingkey` being set to `keyId` provided specifically to renovate can override global `user.signingkey` which may be different.